packages/turbo-workspaces/src/commands/summary/index.ts

```diff
@@ -88,7 +88,7 @@ export async function summaryCommand(directory: SummaryCommandArgument) {
     Object.keys(workspacesByDirectory).forEach((dir, idx) => {
       renderDirectory({
         number: idx + 1,
-        workspaces: workspacesByDirectory[directory],
+        workspaces: workspacesByDirectory[dir],
```

Comment:
Unrelated discovery?

Reply:
Yea this was introduced (by me) as part of https://github.com/vercel/turbo/pull/5810.

I'm going to move this (and one other fix) out of this PR to another one so we can merge that ahead of this PR, incase we need a patch before bun support is fully ready.

Reply:
Moved here: https://github.com/vercel/turbo/pull/5951